native/src/components/Header.tsx

```diff
@@ -126,7 +127,7 @@ const Header = ({
       title={t(title)}
       iconName={iconName}
       onPress={visible ? onPress : () => undefined}
-      style={{ opacity: visible ? 1 : 0 }}
+      color={visible ? theme.colors.textColor : 'transparent'}
```

Comment:
The `style` prop gets flattened, and becomes therefore an immutable object. When tapping on an icon, an animation starts to make the tapped icon partially transparent in [@react-navigation/elements](https://github.com/react-navigation/react-navigation/blob/09d65bcdc908883c8cdd1b93fde19cec4b799ad6/packages/react-native-tab-view/src/PlatformPressable.tsx#L82). With tighter controls on the new RN version, tapping on the icon and thus changing the opacity on the immutable style object, the app crashed.

I worked around that by passing in the `color` prop separately from the `style`.

Reply:
🔧 Please add a comment explaining this to the code :)

Reply:
Sorry, I phrased it poorly, this is not a workaround but what we should have been doing for a long time. RN only started punishing us for it now.